pkg/cloud/ibmcloud/types.go

```diff
@@ -29,4 +30,5 @@ type ActuatorParams struct {
 	Client        client.Client
 	EventRecorder record.EventRecorder
 	Scheme        *runtime.Scheme
+	CoreClient    corev1.CoreV1Interface
```

Comment:
Please clean it if it is not used any more.

Reply:
sorry, I used coreClient to do global search ...